src/lib/AppRegistry.tsx

```diff
@@ -74,13 +74,9 @@ interface ArtistProps {
   isPad: boolean
 }
 
-const Artist: React.SFC<ArtistProps> = track<ArtistProps>(props => {
-  return {
-    context_screen: Schema.PageNames.ArtistPage,
-    context_screen_owner_slug: props.artistID,
-    context_screen_owner_type: Schema.OwnerEntityTypes.Artist,
-  }
-})(props => <ArtistRenderer {...props} render={renderWithLoadProgress(Containers.Artist, props)} />)
+const Artist: React.SFC<ArtistProps> = props => (
+  <ArtistRenderer {...props} render={renderWithLoadProgress(Containers.Artist, props)} />
+)
```

Comment:
Is this change, to remove tracking, intended?

Reply:
Yeah I moved the screentrack here https://github.com/artsy/emission/pull/1977/files#diff-3a6742a6612df63240e3e2279fb6fc86R22 so that I could also track both the `slug` and `internalID`

Reply:
Nice 👍